host/tests/L3_TestCases/dsHdmiIn/dsHdmiIn_test11_GetEDIDInfo_Verify.py

```diff
@@ -26,9 +26,11 @@
 from enum import Enum, auto
 
 dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path))
```

Comment:
Why do we need this os.path.join(dir_path) in all files ?

Reply:
Here the os.path.join seems redundant as dir_path already has full path.
But functionality wise it is not affected